WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostNoSiteToReblogActivity.kt

```diff
@@ -0,0 +1,36 @@
+package org.wordpress.android.ui.reader
+
+import android.R.id
+import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.reader_no_site_to_reblog.*
+import org.greenrobot.eventbus.EventBus
+import org.wordpress.android.R
+
+/*
+ * Serves as an intermediate screen where the user is informed that a site is needed for the reblog action
+ */
+class ReaderPostNoSiteToReblogActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.reader_no_site_to_reblog)
+
+        supportActionBar?.setDisplayShowTitleEnabled(false)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
```

Comment:
I think we can maybe make it more synthetic using `supportActionBar?.apply {...}`; wdyt?

Reply:
Fixed with 8eb14af7c180569716f5aaf8e86f6b55f5ca78e5